### ISSUE TITLE
Final project

### ISSUE THREAD

**Issue by bensoltoff** [uc-cfss/viz-brianp1#4]
Is this your project in its completed form?

**Comment by brianp1** [comment]
Yeah, I am sorry for the delay. I had a number obstacles regarding spatial polygon data frames, trying to create hexagons from scratch, figuring out how to accurately get values assigned to the correct tiles, and deploying my app

**Comment by brianp1** [comment]
Like my dashboard runs perfectly fine, but the second I try to publish it, it says that it could not install this package
